asciidoc/courses/genai-mcp-build-custom-tools-python/modules/1-getting-started/lessons/3c-create-first-server/lesson.adoc

```diff
@@ -0,0 +1,187 @@
+= Create Your First MCP Server 
```

Comment:
I think the language in this lesson could be simplified. There is a lot of "Now", "Lets", "First" use which could be condensed.

The use of "this" throughout the lesson could be confusing. It may not always be obvious what this is referring too.